app/jobs/parse_wdl_workflow.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import sys
+import WDL
+import json
+
+# function names we can parse for Apply expressions
+SELECT_ALL = 'select_all'
+GLOB = 'glob'
+
+# We use this custom source reader to pass in input from stdin
+async def read_stdin(uri, path, importer):
+  if uri != "stdin":
+    raise ValueError("This script only handles input from stdin.")
+  wdl = ""
+  for line in sys.stdin:
+    wdl += line
+
+  return WDL.ReadSourceResult(wdl, uri)
+
+
+def main():
+  # load WDL document
+  doc = WDL.load("stdin", read_source=read_stdin)
+  assert doc.workflow, "No workflow in WDL document"
```

Comment:
Can you create a custom exception here? With a well defined name is easier to be handled or identified in the output by the client that calls this script (e.g. `NoWorkflowFound`). 
In general asserts are used for situations where your code is invalid and should not be used in normal error flows.